src/poll.rs

```diff
@@ -434,11 +431,10 @@ impl Registry {
     /// multiple threads.
     ///
     /// [`event::Source`]: crate::event::Source
-    /// [`struct`]: #
-    /// [`reregister`]: #method.reregister
-    /// [`deregister`]: #method.deregister
-    /// [`poll`]: #method.poll
-    /// [`Token`]: struct.Token.html
+    /// [`poll`]: Poll::poll
+    /// [`reregister`]: Registry::reregister
+    /// [`deregister`]: Registry::deregister
+    /// [`Token`]: Token
```

Comment:
It looks like a lot of the links don't work properly in the documentation, but since this PR is changing these they should be:

```suggested
/// [`poll`]: struct.Poll.html#method.poll
/// [`reregister`]: #method.reregister
/// [`deregister`]: #method.deregister
/// [`Token`]: struct.Token.html
```

Reply:
These are intra docs links. They link e.g. the `Poll::poll` method. These should work, or at least don't give any warning on my machine.

Reply:
So running `cargo doc --open` locally and clicking these links in the `Registry` documentation works? It does not me; it loads a blank page.

Reply:
As discussed on Discord; this is nightly only (https://github.com/rust-lang/rust/issues/63305).